datasets/cmu_mocap/README.md

```diff
@@ -0,0 +1,252 @@
+
+The AMC/ASF files can be parsed using [AMCParser library](https://github.com/CalciferZh/AMCParser). You can check the Usage section in this libary for more information.
+
+- c3d : The C3D format stores 3D coordinate information, analog data and associated information used in 3D motion data capture and subsequent analysis operations. 
+
+You can use [py-c3d library](https://c3d.readthedocs.io/en/stable/) for reading and writing C3D binary files.
+
+- avi and mpg : AVI (Audio Video Interleave) has been the long standing format to save and deliver movies and other video files. An MPG file is a common video file that uses a digital video format standardized by the Moving Picture Experts Group (MPEG)
+
+You can use a video loader library such as [decord](https://github.com/dmlc/decord) to load and process these files.
+
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+All the categories, sub-categories and descriptions use English as their primary language.
+
+## Dataset Structure
+
+### Data Instances
+
+A sample from the CMU MoCap dataset is provided below:
+```
+{
+    'subject_id': 12, 
+    'categories': ['Physical Activities & Sports', 'Locomotion', 'Locomotion', 'Locomotion'], 
+    'subcategories': ['martial arts', 'walking', 'walking', 'walking'], 
+    'descriptions': ['tai chi', 'walk', 'walk', 'walk'], 
+    'motions': {
+        'amc_files': ['~/all_asfamc/subjects/12/12_04.amc', '~/all_asfamc/subjects/12/12_01.amc', '~/all_asfamc/subjects/12/12_03.amc', '~/all_asfamc/subjects/12/12_02.amc'], 
+        'skeleton_file': '~/all_asfamc/subjects/12/12.asf',
+    }
+}
+```
+Depending on the file format, 'motions' field would change. Check the Data Fields section for more information.
+
+### Data Fields
+
+The following fields are common for all file formats of the dataset.
+
+- `subject_id`: Unique ID for each subject in the MoCap dataset.
+- `categories`:  A list containing the category names of all trials.
+- `subcategories`: A list containing the subcategory names of all trials.
+- `descriptions`: A list containing the descriptions of all trials.
+
+The c3d, mpg and avi formats have: 
+- `motions` : A list containing the path to all motion files.
+
+The "asf/amc" config has an additional field :
+- `motions`: {
+        `amc_files`: A list containing the path to all motion files, 
+        `skeleton_file`: path to the ASF file,
+    }
+
+    It is a dictionary containing the motion files and skeleton file.
+
+ASF file is the skeleton file, in the ASF file a base pose is defined for the skeleton that is the starting point for the motion data. More info can be found [here](https://research.cs.wisc.edu/graphics/Courses/cs-838-1999/Jeff/ASF-AMC.html).
```

Comment:
It's better to have this info in the Dataset Preprocessing section

Reply:
Moved to Dataset Preprocessing section.